source/numbers.rst

```diff
@@ -42,6 +42,7 @@ You just type them as literals::
 Questions
 ---------
```

Comment:
I'd put in a question that gets them to discover that when you do division with ints you always get an int as I think thats likely to be a common bug cause